src/probeinterface/probe.py

```diff
@@ -165,15 +177,15 @@ def get_title(self) -> str:
             model_name = self.model_name
             manufacturer = self.manufacturer
             txt = ""
-            if len(name) > 0:
+            if name is not None:
```

Comment:
```suggestion
            name = name if name else ""
```
If you want to keep the old behavior.